olm/build-bundles.sh

```diff
@@ -62,44 +62,55 @@ bundle-clean() {
 }
 
 bundle-build() {
-  local BUNDLE_NAME="$1"
-  local INPUT_DIR="$2"
-  local OUTPUT_DIR="$3"
-
-  mkdir -p "$OUTPUT_DIR"
+  local OPERATOR="$1"
+  local VERSION="$2"
+  local INPUT_DIR="$3"
+  local OUTPUT_DIR="$4"
 
-  # Generate manifests
-  "${OPERATOR_SDK}" generate bundle \
-    --deploy-dir "$INPUT_DIR" \
-    --output-dir "$OUTPUT_DIR" \
-    --package "$BUNDLE_NAME"
+  local BUNDLE_NAME="stackable-${OPERATOR}-operator"
 
-  # For whatever reason the generated CSV drops the spec.relatedImages and spec.install properties
-  cp "${INPUT_DIR}"/manifests/*.clusterserviceversion.yaml "$OUTPUT_DIR/manifests"
+  local CHANNEL="$(echo "$VERSION" | sed 's/\.[^.]*$//')"
 
-  # Generate metadata
-  "${OPERATOR_SDK}" generate bundle \
-    --metadata \
-    --output-dir "$OUTPUT_DIR" \
-    --package "$BUNDLE_NAME" \
-    --channels "stable,$CHANNEL" \
-    --default-channel "$CHANNEL"
-
-  # The annotations generated by the operator sdk do not include the openshift versions.
-  grep com.redhat.openshift "$INPUT_DIR"/metadata/annotations.yaml >> "$OUTPUT_DIR"/metadata/annotations.yaml
+  mkdir -p "$OUTPUT_DIR"
+  cp -r "${INPUT_DIR}/manifests/" "$OUTPUT_DIR/manifests"
+  cp -r "${INPUT_DIR}/metadata/" "$OUTPUT_DIR/metadata"
+
+  cat > "${OUTPUT_DIR}/bundle.Dockerfile" <<BUNDLE_DOCKERFILE
+FROM scratch
+
+# Core bundle labels.
+LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
+LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
+LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
+LABEL operators.operatorframework.io.bundle.package.v1=stackable-${OPERATOR}-operator
+LABEL operators.operatorframework.io.bundle.channels.v1=stable,${CHANNEL}
+LABEL operators.operatorframework.io.bundle.channel.default.v1=${CHANNEL}
+LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.39.1
```

Comment:
is it worth providing that as an argument-with-default, jsut so we don't forget about it?

Reply:
In fact I [removed](https://github.com/stackabletech/stackable-utils/pull/102/commits/f5ed7555eb090b37a453efa8cb3a9ce9cfb4ce07) all metrics labels since we don't need them.

Reply:
Even better! :+1: